std_msgs/Header header
Object3dDesc[] objects
